### DISCUSSION TITLE
Can we add a hint about the rocket emoji?

### DISCUSSION THREAD

**Discussion by charliematters** [mobxjs/mobx#2465] [Ideas]
Congratulations on the release of 6 - I know what I'm going to be doing at work for the next few days.

I was reading the [main page](https://mobx.js.org/README.html) and noticed the rocket emoji on a lot of the links in both the sidebar and the references at the bottom. It didn't seem to have any hover text, and nothing on the page indicated what it was. It was only reading Michael's blog where it's described:
> Also, we marked all non-essential knowledge in the docs with a {🚀} rocket emoji, to make it clear which knowledge is optional.

Now I know this, it's really useful but to someone arriving at the front door with no knowledge of the recent changes, it's a bit of a mystery

**Comment by mweststrate** [comment]
yeah we mentioned it in the docs intro, but I can imagine most people skip that section :) https://mobx.js.org/about-this-documentation.html. I'm not sure if docusaurus has the flexibility to do anything richer

**Reply by charliematters** [reply]
Guilty as charged - I went straight for the new examples!

**Reply by jheeffer** [reply]
Same here, went straight to look up something specific and saw the rockets. Tried to hover on them but nothing popped up. Maybe we can add a tooltip, i.e. alt text? It's .md though... not sure if that's possible.

**Comment by mweststrate** [comment]
PRs are welcome. The docssite can be found in the website/ dir.

**Reply by jheeffer** [reply]
I tried by changing it to `<span title="Advanced feature">🚀</span>`, which worked on content pages, but in the sidebar it's encoded to text (`&lt;` etc). Haven't found a way to disable that. Guess that's not PR-worthy then : ).
![image](https://user-images.githubusercontent.com/1596071/95631576-3e2c8d00-0a84-11eb-97a9-a3bfea42d09b.png)

**Comment by mweststrate** [comment]
Maybe it can be done with css and a ::after pseudo selector?

**Reply by jheeffer** [reply]
Hmmm, not sure how that would work. There's no css selector for content (🚀) and can't add my own classes because html is encoded.
But! We can use javascript : )

PR: https://github.com/mobxjs/mobx/pull/2502
